docs/ai/quickstarts/snippets/mcp-server/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+using System.ComponentModel;
+
+// Create a generic host builder for dependency injection, logging, and configuration.
```

Comment:
Code comments don't get localized. Also, for the long ones, it's probably best to put them on two lines so users don't have to scroll in the code block. But maybe it also makes sense to duplicate the important comments in the Markdown after the snippet as a brief explanation of what the code is doing (and it will be localized).